Source/Libraries/openXDA.DataPusher/DataPusherEngine.cs

```diff
@@ -131,16 +124,7 @@ private static void OnUpdateProgressForInstance(string client, string instance,
 
         #region [ Class Functions ]
 
-        public void Initialize()
-        {
-            using (AdoDataConnection connection = ConnectionFactory())
-            {
-                m_dataPusherSettings = new DataPusherSettings();
-                m_dataPusherSettings.Enabled = connection.ExecuteScalar<bool?>("SELECT Value FROM Setting WHERE Name = 'DataPusher.Enabled'") ?? false;
-                m_dataPusherSettings.OnlyValidFaults = connection.ExecuteScalar<bool?>("SELECT Value FROM Setting WHERE Name = 'DataPusher.OnlyValidFaults'") ?? true;
-                m_dataPusherSettings.TimeWindow = connection.ExecuteScalar<int?>("SELECT Value FROM Setting WHERE Name = 'DataPusher.TimeWindow'") ?? 0;
-            }
-        }
+        public void Initialize() {}
 
```

Comment:
Are we keeping this to maintain downstream compatibility with PQMark or something?
```suggestion
```

Reply:
no idea where this is used. I don't think PQMark is still in use. I just left it in there to avoid having to chase down where it is used.

Reply:
It's not used in openXDA so the only place it could be used is in downstream apps. And I don't know of any that would use `DataPusherEngine` except maybe PQMark, and even then.. I think PQMark should be receiving data, not pushing.